VimeoNetworking/VimeoNetworking/Request.swift

```diff
@@ -223,14 +167,37 @@ public struct Request<ModelType: MappableResponse>
     
     internal func associatedPageRequest(newPath newPath: String) -> Request<ModelType>
     {
+        // Since page response paging paths bake the paging parameters into the path (even if we originally provide them in the body),
```

Comment:
Okay, I suggest adding this additional info in a comment in this file. Most people will want this same background info. ❤️